Sources/Publish/Internal/Collection+ContainsPrefix.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Collection+ContainsPrefix.swift
+//  
+//
+//  Created by Ben Syverson on 2020/01/28.
+//
+
+import Foundation
+
+internal extension Collection where Element: StringWrapper {
```

Comment:
I think this is an unnecessary abstraction, given that it's only being used in a single place.